### PR TITLE
PRIME-2478 Support Multiple Organization for single SA

### PR DESCRIPTION
**Pull request by bergomi02**

### CODE REVIEW

**Comment by neophyte57** [comment]
It seems if I have multiple Organizations, I can only download the Org Agreement for the most recently created Organization.  For the others, I see an error 

image

**Comment by bergomi02** [comment]
> It seems if I have multiple Organizations, I can only download the Org Agreement for the most recently created Organization. For the others, I see an error
> 
> image

it is fixed.

File: prime-dotnet-webapi/Services/OrganizationService.cs [hunk 1]
```diff
@@ -12,6 +12,7 @@
 using Prime.HttpClients.DocumentManagerApiDefinitions;
 using Prime.Models;
 using Prime.ViewModels;
+using Prime.Migrations;
```
Comment (neophyte57) [inline prime-dotnet-webapi/Services/OrganizationService.cs:15]: Import not necessary

Comment (bergomi02) [reply prime-dotnet-webapi/Services/OrganizationService.cs]: it is fixed.

File: prime-angular-frontend/src/app/modules/site-registration/pages/site-management-page/site-management-page.component.ts [hunk 2]
```diff
@@ -193,23 +207,32 @@ export class SiteManagementPageComponent implements OnInit {
...
+
           this.organizationSitesExpiryDates = organization.sites
             .reduce((expiryDates: string[], site: Site) => {
               return (site.status === SiteStatusType.EDITABLE && !!site.approvedDate)
                 ? [...expiryDates, Site.getExpiryDate(site)]
                 : expiryDates;
             }, []);
-          return this.organization = organization;
+          this.organizations = organizations;
+          return this.organization = organizations.find((org) => org.id === this.organizationService.organization.id);
```
Comment (neophyte57) [inline prime-angular-frontend/src/app/modules/site-registration/pages/site-management-page/site-management-page.component.ts:235]: This seems to work but shouldn't this be `organizations.find((org) => org.id === organization.id);` instead?

**Review by neophyte57 — APPROVED** [review]
🎉 Great job, such an elegant implementation! 💯

**Pull request merged**